thoth/storages/graph/postgres.py

```diff
@@ -453,7 +455,7 @@ def _do_software_environment_analyses_listing(
         self,
         software_environment_name: str,
         start_offset: int,
-        count: int,
+        count: Optional[int],
```

Comment:
Could you point me to docs where `limit` can be set to `None`? I tried to look up this behaviour, but failed:

[1] https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.limit
[2] https://github.com/sqlalchemy/sqlalchemy/blob/a54527fb0ef71aca0be9614617f143da27e03f22/lib/sqlalchemy/sql/selectable.py#L2669-L2670

Reply:
[3] https://stackoverflow.com/questions/12836194/how-do-i-remove-a-limit-that-is-already-applied-to-a-query-object

I found this

Reply:
And also, default value of limit is `None`
[4] https://github.com/sqlalchemy/sqlalchemy/blob/a54527fb0ef71aca0be9614617f143da27e03f22/lib/sqlalchemy/sql/selectable.py#L2457-L2462

Reply:
I'm not that happy about merging officially undocumented functionality as it might break with updates. As this is sqlalchemy, let's keep eyes closed and assume their api is stable.

Reply:
> And also, default value of limit is `None`
> [4] https://github.com/sqlalchemy/sqlalchemy/blob/a54527fb0ef71aca0be9614617f143da27e03f22/lib/sqlalchemy/sql/selectable.py#L2457-L2462

But checked before applying it:

https://github.com/sqlalchemy/sqlalchemy/blob/a54527fb0ef71aca0be9614617f143da27e03f22/lib/sqlalchemy/sql/selectable.py#L2482-L2483